imap_processing/swe/utils/swe_utils.py

```diff
@@ -0,0 +1,131 @@
+import collections
+from enum import IntEnum
+
+import numpy as np
+import xarray as xr
+
+from imap_processing import cdf_utils
+from imap_processing.swe import swe_cdf_attrs
+
+
+class SWEAPID(IntEnum):
+    """Create ENUM for apid.
+
+    Parameters
+    ----------
+    IntEnum : IntEnum
+    """
+
+    SWE_APP_HK = 1330
+    SWE_EVTMSG = 1317
+    SWE_CEM_RAW = 1334
+    SWE_SCIENCE = 1344
+
+
+def get_descriptor(apid: int):
+    """Get descriptor for the output filename.
+
+    Parameters
+    ----------
+    apid : int
+        Packet apid
+
+    Returns
+    -------
+    str
+        Filename description
+    """
+    if apid == SWEAPID.SWE_APP_HK.value:
+        description = "hk"
+    elif apid == SWEAPID.SWE_EVTMSG.value:
+        description = "evtmsg"
+    elif apid == SWEAPID.SWE_CEM_RAW.value:
+        description = "cemraw"
+    elif apid == SWEAPID.SWE_SCIENCE.value:
+        description = "sci"
+    else:
+        return None
+    return description
```

Comment:
Use a simple lookup table instead?
```suggestion
filename_descriptors = {SWEAPID.SWE_APP_HK: "hk", ...}
```

Reply:
will do that.

Reply:
Remove the function altogether and just use the lookup dictionary directly? The function here is just wrapping the dict lookup, so if you really wanted to do a method `()` call instead of an index `[]` call, I think you might be able to do: `get_descriptor = {...}.get`

Reply:
so if we remove function, `filename_descriptors` will be global variable in the `swe_utils.py`?

Reply:
yes, it would be a module level variable.

```python
swe_utils.filename_descriptors[1234]
# versus
swe_utils.get_descriptor(1234)
```

Reply:
ok. That makes sense now.